build.cmd

```diff
@@ -233,7 +236,8 @@ if %__BuildNative% EQU 1 (
     echo %__MsgPrefix%Regenerating the Visual Studio solution
 
     pushd "%__IntermediatesDir%"
-    call "%__SourceDir%\pal\tools\gen-buildsys-win.bat" "%__ProjectDir%" %__VSVersion% %__BuildArch% %__BuildJit32%
+    set __ExtraCmakeArgs="-DCLR_CMAKE_TARGET_OS=%__BuildOs%" "-DCLR_CMAKE_PACKAGES_DIR=%__PackagesDir%" "-DCLR_CMAKE_PGO_INSTRUMENT=%__PgoInstrument%"
```

Comment:
@janvorli I believe it's being set as a default [here](https://github.com/dotnet/coreclr/blob/master/CMakeLists.txt#L220). It's being set in the other location because host != target in the cross build scenario. Should I still set it here?